package.json

```diff
@@ -7,7 +7,9 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "build:prod": "SET NODE_ENV=production&& ng build --prod",
+    "build:prod:linux": "NODE_ENV=production ng build --prod"
```

Comment:
Can we put "NODE_ENV=production ng build --prod" on "build" script? (which will be defaulted to prod soon by the CLI, and hopefully also the NODE_ENV to production)

I would suggest not to have an entry for windows because AFAIK. there is also $env:NODE_ENV="production" for PS. We'll use cross-env if needed.

Reply:
You right @chihab , I saw this breaking change in Angular backlog https://github.com/angular/angular-cli/issues/14471

So, bcz `NODE_ENV=production ng build --prod` works only on Linux and most of users are using Windows, we need to inform them (on Readme for ex) to run a specific prod build script for windows (ex `npm run build:prod:win`)

_FYI: `$env:NODE_ENV="production` is for PS and `SET NODE_ENV=production is for CMD`, so it's the same thing for windows_ 
_FYI: PowerShell is cross-env now, but it require Linux/Mac users to install it_ 😉

Reply:
You're right, we should document that for Windows users.
The thing is that we almost never have to run ng build --prod locally so this is mainly for CI which would surely be on a Linux.
If someone wants to do it locally, he should be aware that he has to run a specific command for his use case.
If he's on Unix NODE_ENV=, on Windows and using CMD (by default on VScode) SET NODE_ENV and if using PowerShell $env:NODE_ENV... we don't usually deal with these things on npm scripts.
But let's put that on the README just in case...

Reply:
So what do you suggest now.

1 - keep the two script helpers or delete them ? :
`build:prod` (its good to change it to build:prod:win)
`build:prod:linux`

2 - Change `build` script to `production ng build --prod` ?

Reply:
I suggest ` "build" : "NODE_ENV=production ng build --prod"`

What I hope is that one day (with some cli version) we would have: ` "build" : "ng build"` which would be exactly the same

Reply:
OK, thats clear the default build behavior.

and What about the script helpers (build:prod:linux and build:prod:win), do you suggest to delete them or keep them ?

Reply:
I suggest at least to keep the one for windows users (build:prod:win), just in case some windows user want to run prod build

Reply:
Just the "build" script please.

Reply:
Okey